app/controllers/application_controller.rb

```diff
@@ -1,10 +1,18 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.